scalableminds/webknossos
# Add nd support for tiff export job

This PR enables tiff export for nd datasets.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz -> not deployed as corresponding worker version is needed.

### Steps to test:
- Needs to be tested together with the worker of PR: https://github.com/scalableminds/voxelytics/pull/3667
- Setup wk and worker locally
- Test 3d dataset tiff export:
  - Choose your favourite 3D dataset, create an annotation and create a bbox
  - Open the download modal  e.g. via the bbox dropdown menu and try to export the bbox as ome tiff and tiff stack zip
- Test new nd dataset tiff export:
  - Choose your favourite ND dataset, create an annotation and create a bbox
  - Open the download modal  e.g. via the bbox dropdown menu and try to export the bbox as ome tiff and tiff stack zip
- All downloads should work
 
### TODOs:
- [ ] Nothing, hopefully :)

### Issues:
- fixes [#7279](https://github.com/scalableminds/webknossos/issues/7279)

------
(Please delete unneeded items, merge only when none are left open)
- [ ] Considered [common edge cases](../blob/master/.github/common_edge_cases.md) - 2d datasets maybe?
